manifest/pack.go

```diff
@@ -98,6 +100,26 @@ func (mp mpack) Pull(ctx context.Context, opts ...pack.PullOption) error {
 	return mp.manifest.Provider.PullManifest(ctx, mp.manifest, opts...)
 }
 
+func (mp mpack) Delete(ctx context.Context, version string) error {
+	cacheDir := mp.manifest.mopts.cacheDir
+	localAvailablePackages, err := os.ReadDir(cacheDir)
+	if err != nil {
+		return err
+	}
+	for _, localPackage := range localAvailablePackages {
+		if strings.Contains(localPackage.Name(), mp.Name()) && strings.Contains(localPackage.Name(), mp.Version()) {
+			if !strings.HasSuffix(cacheDir, "/") {
+				cacheDir += "/"
+			}
+			err = os.Remove(cacheDir + localPackage.Name())
```

Comment:
You can use functions from the `filepath` package to work with paths

Reply:
Done 👍🏽